example-serverless-app-reuse/reuse-auth-only.yaml

```diff
@@ -64,10 +116,19 @@ Resources:
             Id: dummy-origin
             CustomOriginConfig:
               OriginProtocolPolicy: match-viewer
-          - DomainName: change.this.to.your.origin
+          - DomainName: !Sub "${ApplicationBucket}.s3.amazonaws.com"
             Id: protected-origin
-            CustomOriginConfig:
-              OriginProtocolPolicy: match-viewer
+            S3OriginConfig:
+              OriginAccessIdentity: !Sub "origin-access-identity/cloudfront/${CloudFrontOriginAccessIdentity}"
+        CustomErrorResponses:
+            - ErrorCode: 404
+              ResponseCode: 200
+              ResponsePagePath: /index.html
+            - ErrorCode: 403
```

Comment:
We should remove this one (the 403) - read #25 to understand why

Reply:
Makes sense with the combination of additional bucket permission for CFront and the index default